sigmajs/sigmajs.d.ts

```diff
@@ -149,7 +163,7 @@ declare module SigmaJs{
 
     interface SigmaConfigs {
         container?: Element;
-        graph?: Graph;
+        graph?: GraphData;
```

Comment:
why this change? graph object has many properties, this will break all the graph related stuff